### ISSUE TITLE
[Feature]: Add support for new Zotero item types

### ISSUE THREAD

**Issue by marcschulder** [retorquere/zotero-better-bibtex#2496]
### Debug log ID

T3EKEWWA-refs-euc

### What happened?

Zotero v6.0.24  added new item types _dataset_ and _standard_. BBT currently maps these to _misc_ when exporting to BibLaTeX. As BBT otherwise supports the _dataset_ reference type, it should use it instead. (BibLaTeX also has a non-standard type _standard_, so it would probably be appropriate to remap that as well.)

**Comment by retorquere** [comment]
Does this do what you want?

**Comment by marcschulder** [comment]
Works great for _dataset_. Doesn't yet work for _standard_.

**Comment by retorquere** [comment]
I see. What would you propose `standard` exports to?

@njbart?

**Comment by retorquere** [comment]
There's only a `dataset` entry in `T3EKEWWA-refs-euc` BTW.

**Comment by marcschulder** [comment]
Sorry about the missing example in the debug, `dataset` was my personal use case and I only started thinking about `standard` when I wrote up the bug report.
According to the biblatex doc, there is a non-standard reference type `standard` which I believe represents the same concept as the new Zotero item type. As it is non-standard, it will be treated like misc by default and expects the same set of fields, so the current BBT behaviour isn't that bad, but mapping to `@standard` might still be good for users who do define their custom formatting for it (similar to e.g. `Video Recording` mapping to `@video`)
